### ISSUE TITLE
SwiftUI Warning: Accessing StateObject's object without being installed on a View. This will create a new instance each time.

### ISSUE THREAD

**Issue by larryonoff** [kean/Nuke#763]
This happens in `LazyImage` for the latest Nuke 12.4 on any iOS 17.

```swift
    public func onStart(_ closure: @escaping (ImageTask) -> Void) -> Self {
        map { $0.viewModel.onStart = closure }
    }
```

**Comment by kean** [comment]
Fixed in [Nuke 12.6](https://github.com/kean/Nuke/releases/tag/12.6.0). Thanks for the report!

**Comment by larryonoff** [comment]
@kean thanks for the fix!
